hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This patch fixes type-checking errors in our vendored prety-printer,
```

Comment:
Whoops, missed a typo here: _prety_ → _pretty_.

Reply:
No worries, it's already fixed in #1375 :smile: